gateway-ha/src/test/java/io/trino/gateway/ha/handler/TestQueryIdCachingProxyHandler.java

```diff
@@ -61,6 +66,44 @@ public void testExtractQueryIdFromUrl()
                 .isNull();
     }
 
+    @Test
+    public void testQueryIdFromKill()
+            throws IOException
+    {
+        HttpServletRequest req = Mockito.mock(HttpServletRequest.class);
+
+        String query = "CALL system.runtime.kill_query(query_id ==> '20200416_160256_03078_6b4yt', message ==> 'If he dies, he dies')";
```

Comment:
`==>` should be `=>`. Also, please add other cases:
* Without message argument `CALL system.runtime.kill_query(query_id => '')`
* Without name arguments, `CALL system.runtime.kill_query('...', '...')`

Reply:
The extra test cases identified that the logic was relying on the named `query_id` argument, so I have corrected the logic in `extractQueryIdIfPresent` to handle these variations. Thanks!